src/bindings/html/dom.js

```diff
@@ -49,7 +49,7 @@ function getTranslatables(element) {
     nodes, element.querySelectorAll('*[data-l10n-id]'));
 }
 
-export function translateDocument(doc, langs) {
+export function translateDocument(view, doc, langs) {
```

Comment:
why do you need both, view and doc? You should be able to get doc out of view object, right?